SampleGoGrpc/SimpleMath/grpc.go

```diff
@@ -0,0 +1,259 @@
+//Generated by GRPC Go plugin
+package SimpleMath
+
+import "github.com/google/flatbuffers/go"
```

Comment:
As of now all the services are generated in the same file grpc.go, should generator each service in a separate file just like how the each table has its own file?